app/controllers/shops_controller.rb

```diff
@@ -20,4 +43,15 @@ def load_shop
     flash[:danger] = t("shops.not_found")
     redirect_to root_url
   end
+
+  def shop_params
+    params.require(:shop).permit(:name, :description, :user_id)
```

Comment:
tào lao tui chém chừ, sao ko lấy từ current user mà phải truyền user_id vào thế
```suggestion
    params.require(:shop).permit(:name, :description)
```